iquip/apps/builder.py

```diff
@@ -406,15 +408,15 @@ def value(self) -> Dict[str, Any]:
 
     @pyqtSlot(QAbstractButton)
     def scanTypeClicked(self, selectedButton: QAbstractButton):
-        """Switches current scan widget at stacked layout in _ScanEntry.
+        """Switches current scan widget in the stacked layout.
         
-        Once the checked button at radiobutton group clicked, this is called.
+        Once a scan type button at radiobutton group clicked, this is called.
 
         Attributes:
-            selectedButton: A QRadioButton that is clicked.
+            selectedButton: The clicked QRadioButton.
         """
-        selectedEnum = _ScanEntry.ScanType(self.scanButtonGroup.id(selectedButton))
-        self.stackedWidget.setCurrentWidget(self.scanWidgets[selectedEnum.name])
+        selectedType = _ScanEntry.ScanType(self.scanButtonGroup.id(selectedButton))
```

Comment:
You renamed the same variable in `_ScanEntry.__init__()` to `selectedScanType`, hence please rename this to the same.